assets/src/edit-story/components/canvas/editElement.js

```diff
@@ -34,7 +34,7 @@ import { useUnits } from '../../units';
 const Wrapper = styled.div`
 	${elementWithPosition}
 	${elementWithSize}
-	${elementWithRotation}
+	${({ hasEditModeMovable }) => !hasEditModeMovable && elementWithRotation}$
```

Comment:
Why hide the rotation? And also why the tail `$`?

Reply:
Oops, the `$` should be removed.

Hiding the rotation since the Moveable in this PR is targeting the inner Text Edit directly instead of the general wrapper. However, the rotation is on the general wrapper -- this causes issues for Moveable.

Actually, tested it out a bit more yesterday and this PR generally has some weirdness due to targeting the inner Text Edit.

The other test PR makes more sense actually: https://github.com/google/web-stories-wp/pull/2699

I think we could close this PR in favor of trying out that one ^.